packages/frontend2/src/app/_components/nav/nav-sidebar-wrapper.tsx

```diff
@@ -20,7 +20,7 @@ export const NavSideBarWrapper = ({
   )
 
   useEffect(() => {
-    let timeout: NodeJS.Timeout
+    let timeout: ReturnType<typeof setTimeout>
```

Comment:
I guess we want to leave a TODO here because this is clearly an issue with typing

Reply:
If I am correct its a difference between types for `setTimeout` for `node` and `window` environments. So I guess the safest way to type it is the one above